notebooks/advanced/Compare_Multiple_Models/Compare_Multiple_Models.ipynb

```diff
@@ -517,7 +517,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.5"
+   "version": "3.7.3"
   }
```

Comment:
I suppose this change to unrelated notebook file was not intended.

Reply:
Yes you are right. I did not mean to change anything in that notebook. I opened it to look at how something had been implemented to get an idea of how to do it on the notebook I am currently working on. It must have updated the python version when I did a pip install --upgrade on my local branch, although I am not exactly sure how.